src/hb-ot-glyf-table.hh

```diff
@@ -267,15 +267,99 @@ struct glyf
 			       CompositeGlyphHeader::Iterator *composite /* OUT */) const
     {
       unsigned int start_offset, end_offset;
-      if (!get_offsets (glyph, &start_offset, &end_offset))
+      if (!get_offsets (glyph, /* trim */ false, &start_offset, &end_offset))
```

Comment:
Ew. No. Trim in get_offsets is horrible API...  What does it even mean?!

Reply:
Sorry, was trying to figure out why my glyphs weren't the same size as fonttools and stuck that in temporarily. Is now it's own API.